monkestation/code/modules/mech_comp/objects/_object.dm

```diff
@@ -16,14 +16,19 @@
 	inputs = list()
 	update_icon_state()
 
-	MC_ADD_CONFIG(MC_CFG_UNLINK_ALL, unlink_all)
-	MC_ADD_CONFIG(MC_CFG_UNLINK, unlink)
 	MC_ADD_CONFIG(MC_CFG_LINK, add_linker)
+	MC_ADD_CONFIG(MC_CFG_UNLINK, unlink)
+	MC_ADD_CONFIG(MC_CFG_UNLINK_ALL, unlink_all)
 
 /obj/item/mcobject/Destroy(force)
 	QDEL_NULL(interface)
 	return ..()
 
+/obj/item/mcobject/examine(mob/user)
+	. = ..()
+	. += span_notice("You can left-click with a multitool to put up a list of options available for linking.")
+	. += span_notice("You can right-click with a multitool to quickly start linking the device as an output.")
```

Comment:
should prolly also add these as screentips

Reply:
done